editor/project_manager.cpp

```diff
@@ -2084,6 +2100,17 @@ void ProjectManager::shortcut_input(const Ref<InputEvent> &p_ev) {
 			} break;
 		}
 
+		if (Input::get_singleton()->is_key_pressed(Key::CMD_OR_CTRL)) {
+			if (k->get_keycode() == Key::A) {
```

Comment:
Can we use an action instead? (Well, two actions, one for select and one for deselect.) We should have access to editor shortcuts in the PM.

Reply:
I'm not sure if it's really worth adding configurable actions for those standard shortcuts (we already have a few of them here, such as <kbd>Ctrl + F</kbd> just above).

Either way, if we start to do that, we should probably add `ui_select_all` and `ui_deselect_all` (or `ui_select_none`), so that we can use the same shortcut in FileDialog, EditorFileDialog and so on. There's a lot of locations where these shortcuts could be implemented.